Sources/Frontend/Commands/ScanCommand.swift

```diff
@@ -136,6 +142,10 @@ struct ScanCommand: FrontendCommand {
             Logger().warn("The option '--external-encodable-protocols' is deprecated, use '--external-codable-protocols' instead.")
         }
 
+        if baseline != nil, writeBaseline != nil {
```

Comment:
In the SwiftLint implementation we allow both together, so that you could have a standard baseline that you're filtering against, but still be writing a baseline for the current build. I don't have a good use case for that, but in the SwiftLint case we read the baseline first, and write it later, so there was no need for any restriction - I'm not sure what the ordering is here.

If the baseline file to read does not exist, SwiftLint does treat that as an error, unless `--write-baseline` is also there, and points to the same file - for the very rare use case where you've set everything up on CI, and just need to do a `--write-baseline` occasionally.

Reply:
Removed the restriction 👍